elasticapm/contrib/flask/__init__.py

```diff
@@ -131,6 +132,23 @@ def init_app(self, app, **defaults):
         else:
             logger.debug("Skipping instrumentation. INSTRUMENT is set to False.")
 
+        @app.context_processor
+        def rum_tracing():
+            """
+            Adds APM related IDs to the context used for correlating the backend transaction with the RUM transaction
+            """
+            transaction = get_transaction()
+            if transaction.trace_parent:
+                return {
+                    "apm": {
+                        "trace_id": transaction.trace_parent.trace_id,
+                        "span_id": lambda: transaction.ensure_parent_id(),
```

Comment:
could this be `transaction.ensure_parent_id` instead of a lambda?

Reply:
Yes! Good catch! I'll also add a comment to avoid that me + 6 months thinks present me was an idiot and forgot `()`